ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/methods/engine/EngineForkchoiceUpdatedV2.java

```diff
@@ -71,6 +71,6 @@ protected Optional<JsonRpcErrorResponse> isPayloadAttributesValid(
 
   @Override
   protected RpcErrorType getInvalidPayloadError() {
```

Comment:
methods should be renamed, this is confusing/misleading

Reply:
refactored this